db/models/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require('../config/config.json')[env];
+
+const db = {};
+
+let sequelize;
```

Comment:
I would suggest renaming this to differentiate it more easily from the uppercase `Sequelize`